core/src/main/resources/defaults/blueprints/cdp-opdb-720.bp

```diff
@@ -225,7 +263,9 @@
           "hbase-GATEWAY-BASE",
           "hdfs-GATEWAY-BASE",
           "knox-KNOX-GATEWAY-BASE",
-          "yarn-GATEWAY-BASE"
+          "yarn-GATEWAY-BASE",
+          "hue-HUE_SERVER-BASE",
+          "hue-HUE_LOAD_BALANCER-BASE"
```

Comment:
If we only have one Hue server, do we still need the Hue LB?

Reply:
Short answer: Yes
Long answer: As we discussed Josh, CM doesn't have an issue with having just one Hue Server neither does Hue have any problem bringing up a single server without a load balancer. But the **Hue Web UI via KNOX** wouldn't be accessible as it depends on the load balancer to be up. So, yes.